eng/_util/cmd/run-builder/systemfips_linux.go

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+package main
+
+import (
+	"log"
+	"os"
+)
+
+// enableSystemWideFIPS enables Mariner and Azure Linux 3 process-wide FIPS mode.
+func enableSystemWideFIPS() (restore func(), err error) {
+	// FIPS mode is enabled if OPENSSL_FORCE_FIPS_MODE is set, regardless of the value.
+	_, ok := os.LookupEnv("OPENSSL_FORCE_FIPS_MODE")
+	if ok {
+		log.Println("FIPS mode already enabled.")
+		return nil, nil
+	}
+
+	env("OPENSSL_FORCE_FIPS_MODE", "1")
+	log.Println("Enabled Mariner and Azure Linux 3 FIPS mode.")
```

Comment:
`env` will have just logged a message, but it doesn't hurt to make this a bit more explicit too.

```suggestion
	log.Println("Enabled Mariner and Azure Linux 3 FIPS mode (OPENSSL_FORCE_FIPS_MODE).")
```